walk-in-interview/src/main/java/com/google/job/servlets/JobServlet.java

```diff
@@ -65,26 +64,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) {
     /** Handles the PATCH request from client. */
     public void doPatch(HttpServletRequest request, HttpServletResponse response) {
         try {
-            // Gets the target job post id from the form
-            String jobId = ServletUtils.getStringParameter(request, JOB_ID_FIELD, /* defaultValue= */ "");
-
-            // Gets job post from the form
+            // Gets job post (with cloud firestore id) from the form
             Job updatedJob = parseRawJobPost(request);
+            String jobId = updatedJob.getJobId();
 
             // Stores job post into the database
             updateJobPost(jobId, updatedJob);
```

Comment:
do we need to check if user has the authorization to update this?

Reply:
this will be checked in `setJob` in JobsDatabase once the account is involved